webui/react/src/hooks/useTheme.ts

```diff
@@ -51,13 +52,14 @@ const updateAntDesignTheme = (path: string) => {
  * and storybook Theme decorators and not individual components.
  */
 export const useTheme = (): ThemeHook => {
+  const currentTheme = (() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
   const [ branding, setBranding ] = useState(BrandingType.Determined);
-  const [ mode, setMode ] = useState(() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
-
+  const [ mode, setMode ] = useState(currentTheme);
+  const [ scheme, setScheme ] = useState<DarkLight>(currentTheme);
   const theme = useMemo(() => themes[branding][mode], [ branding, mode ]);
```

Comment:
**chore:** this `useMemo` is going to need factor in `scheme`/`systemMode`. The idea is that if the `mode` is set to `Auto` then it needs to use `scheme`/`systemMode`. If `systemMode` is also `Auto` then to keep it simple we can default to `Light`. But interestingly enough, in Mac, if it is `Auto`, it will [change through the day](https://support.apple.com/guide/mac-help/use-a-light-or-dark-appearance-mchl52e1c2d2/mac#:~:text=You%20can%20use%20a%20light,System%20Preferences%2C%20then%20click%20General%20.&text=Select%20Light%2C%20Dark%2C%20or%20Auto.).